shared/src/types/server-error-response.type.ts

```diff
@@ -1,4 +1,4 @@
-import { type ServerErrorType } from '~/enums/enums.js';
+import { type ServerErrorType } from 'shared/src/enums/enums.js';
```

Comment:
Such path will break the production server

Reply:
okay i will change it back